sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -1,19 +1,19 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 12.0.0-beta.5 (2020-11-13)
 
 ### Breaking Changes
 
-* Move header `azure/storage/files/datalake/datalake.hpp` to `azure/storage/files/datalake.hpp`
+* Moved header `azure/storage/files/datalake/datalake.hpp` to `azure/storage/files/datalake.hpp`
+* Moved returning model types and related functions in `Azure::Storage::Files::DataLake` to `Azure::Storage::Files::DataLake::Models`, and other code into `Azure::Storage::Files::DataLake::Details`.
+* Renamed `Azure::Storage::Files::DataLake::ServiceClient` to `Azure::Storage::Files::DataLake::DataLakeServiceClient`.
```

Comment:
It's interesting that we settled on this, i.e. adding a redundant name to the type.

I suspected callers to do `DataLake::ServiceClient` and now that would look like: `DataLake::DataLakeServiceClient`.

It's fine though. If the user cares, they can move `::DataLake` into the using. I just don't know to which types we would add such prefixes. Definitely, not all of them, right?

Reply:
The original thought of calling everything `ServiceClient` was to avoid duplication. However, it is also true that by looking at `ServiceClient`, customer not knowing the service type is weird.
I'm leaning towards duplication for clarity, but not strongly.